app/experimenter/experiments/serializers.py

```diff
@@ -35,6 +35,22 @@ def to_representation(self, obj):
             return obj
 
 
+class VariantsListSerializer(serializers.ListSerializer):
+
+    def to_representation(self, data):
+        data = super().to_representation(data)
+        if data == []:
+            blank_variant = {}
+            for field in self.child.fields:
+                blank_variant[field] = None
+                control_blank_variant = blank_variant.copy()
```

Comment:
Heh, I see what you did here and that's clever but it also wasn't obvious at first glance, just create an empty one for each branch and explicitly set the field on each one ;)